api/v1beta1/cluster_types.go

```diff
@@ -332,6 +332,11 @@ type Topology struct {
 	// The name of the ClusterClass object to create the topology.
 	Class string `json:"class"`
 
+	// The namespace of the ClusterClass object to create the topology.
+	//
+	// +optional
```

Comment:
Q: do we care about differentiating between not set and set to empty string?
(I think not, just double checking)